hail/python/hailtop/utils/utils.py

```diff
@@ -111,12 +112,14 @@ def digits_needed(i: int) -> int:
     return 1 + digits_needed(i // 10)
 
 
-def grouped(n: int, ls: List[T]) -> Iterable[List[T]]:
+def grouped(n: int, ls: Iterable[T]) -> Iterable[List[T]]:
```

Comment:
Could you leave a comment to replace this with [itertools.batched](https://docs.python.org/3/library/itertools.html#itertools.batched) when we (one day hopefully) hit 3.12?

Reply:
done